frontend/js/src/artist-similarity/src/Data.tsx

```diff
@@ -0,0 +1,168 @@
+import React, {useState, useEffect} from "react";
+import Graph from "./Graph";
+import Input from "./Input";
+
+type ColorType = {
+    red: number;
+    green: number;
+    blue: number;
+    opacity: number;
+    mixed: string;
+}
+
+type ArtistType = {
+    artist_mbid: string;
+    name: string;
+    comment: string;
+    type: string;
+    gender: string;
+    score?: number;
+    reference_mbid?: string;
+}
+    
+type MarkupResponseType = {
+    data: string;
+    type: "markup";
+}
+
+type DatasetResponseType = {
+    columns: Array<string>;
+    data: Array<ArtistType>;
+    type: "dataset";
+}
+
+type ApiResponseType = MarkupResponseType | DatasetResponseType;
+
+const Color = (red: number, green: number, blue: number): ColorType => {
+    return {
+        red: red,
+        green: green,
+        blue: blue,
+        opacity: 1.0,
+        mixed: "rgba(" + red + "," + green + "," + blue + "," + 1.0 + ")",
+    }
+}
+
+const changeOpacity = (color: ColorType, opacity: number): ColorType => {
+    return {
+        red: color.red,
+        green: color.green,
+        blue: color.blue,
+        opacity: opacity,
+        mixed: "rgba(" + color.red + "," + color.green + "," + color.blue + "," + opacity + ")",
+    }
+}
+
+const colorGenerator = (): ColorType => {
+    var red = Math.floor(Math.random() * 200);
```

Comment:
We'll get results that are more pleasing to the human eye if we use the hsv_to_rgb function here:

import colorsys
import random

red, green, blue = hsv_to_rgb(random.random(), 1.0, .9)
then convert each from 0->1.0 to 0 -> 255.

Reply:
Wow, what a dumbass. @MonkeyDo pointed out that this is javascript, not python. Lol, sorry. Too focused on my current project, sorry. :)

Reply:
But this ain't python :D

However we do have some color-manipulation library already installed in the repo called tinycolor2: https://bgrins.github.io/TinyColor/

Reply:
Great -- tinycolor("hsv(Math.random(), 100%, 90%)"); should give the results we're looking for.

Reply:
Do we need to multiply random with 360 to generate random color.
```
tinycolor("hsv(" + Math.random() * 360 + ", 100%, 90%)");
```

Reply:
Yes, quite likely.